### PR TITLE
Fixed Ctrl key keyboard interface on macOS

### PR DESCRIPTION
**Pull request by omichel**
A user reported a bug on macOS: when pressing the <kbd>Ctrl</kbd> key in the 3D view together with another key, it was not taken into account by the keyboard API. This PR fixes that bug.

### CODE REVIEW

**Comment by omichel** [comment]
There were actually several problems:
1. The <kbd>Ctrl</kbd> and <kbd>Meta</kbd> keys are [swapped](https://doc.qt.io/qt-5/macos-issues.html#special-keys) on mac OS.
2. The `event->text()` value appeared to be an empty string in some cases, like <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>1</kbd> on mac OS.
3. The latest keyboard change was appended to the list of changes whereas it seems better to prepend it as only the first value is read by the controller.

**Comment by stefaniapedrazzi** [comment]
:eyes:

File: docs/reference/changelog-r2020.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 Released on XXX YYY, 2020.
 
   - Bug fixes
+    - Fixed handling of <kbd>Ctrl</kbd> key on mac OS from the keyboard API ([#2265](https://github.com/cyberbotics/webots/pull/2265)).
```
Comment (stefaniapedrazzi) [inline docs/reference/changelog-r2020.md:7]: minor suggestions to be more consistent with the rest of the changelog entries
```suggestion
    - Fixed handling of <kbd>Ctrl</kbd> key on macOS from the [Keyboard](keyboard.md) API ([#2265](https://github.com/cyberbotics/webots/pull/2265)).
```

**Review by stefaniapedrazzi — APPROVED** [review]
The fix is fine.
I would just wait the result of the tests before merging.

**Pull request merged**